### PR TITLE
FEM-2612 - ad display container is not removed in case destroy happens during ad playback

### PR DESCRIPTION
**Pull request by giladna**
fix FEM-2600 regression

### CODE REVIEW

**Review by GouravSna — APPROVED** [review]

File: imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java [hunk 1]
```diff
@@ -509,8 +509,9 @@ private void clearAdLoadingInBackground() {
 
     @Override
     protected void onDestroy() {
-        log.d("IMA Start onDestroy");
-        boolean adManagerInitialized = (adsManager != null); // FEM-2600
+        log.d("IMA Start onDestroy lastAdEventReceived = " + lastAdEventReceived);
+        boolean adManagerInitialized = (adsManager != null || lastAdEventReceived != null); // FEM-2600
```
Comment (OrenMe) [inline imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java:513]: Does this relate to the black screen issue? Why is this needed?

Comment (OrenMe) [reply imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java:513]: Also, when is it possible that we don’t have adsManager but we have a last ad event revived from the ad? Is it possible?

Comment (giladna) [reply imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java:513]: yes
no such option

Comment (OrenMe) [reply imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java:513]: So why did you add this additional check?

Comment (giladna) [reply imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java:513]: this is comming from another flow in app
not from destroy the activity but destroying the player programatically

@OrenMe 
we have to give patch for this as well

**Review by noamtamim — APPROVED** [review]

**Review by OrenMe — APPROVED** [review]
